src/modules/tas_studio/editor/mod.rs

```diff
@@ -1418,6 +1436,52 @@ impl Editor {
         Ok(())
     }
 
+    fn tick_side_strafe_yawspeed_adjustment(
+        &mut self,
+        mouse: MouseState,
+        keyboard: KeyboardState,
+    ) -> eyre::Result<()> {
+        let Some(adjustment) = &mut self.side_strafe_yawspeed_adjustment else {
+            return Ok(());
+        };
+
+        let bulk_idx = self.selected_bulk_idx.unwrap();
+        let (bulk, first_frame_idx) =
+            bulk_and_first_frame_idx_mut(&mut self.branches[self.branch_idx].branch.script)
+                .nth(bulk_idx)
+                .unwrap();
+
+        let yawspeed = bulk.constant_yawspeed_mut().unwrap();
+
+        if !mouse.buttons.is_right_down() {
+            if !adjustment.changed_once {
+                self.side_strafe_yawspeed_adjustment = None;
+                return Ok(());
+            }
+
+            let op = Operation::SetYawspeed {
+                bulk_idx,
+                from: adjustment.original_value,
+                to: *yawspeed,
+            };
+            self.side_strafe_yawspeed_adjustment = None;
+            return self.store_operation(op);
+        }
+
+        // delta scalar might need being a lot higher but there's +ALT1 adjustment anyway.
```

Comment:
I haven't tried this, but you can change the scale here to feel right. So that by default it's reasonable, and you might occasionally need shift or alt1. You shouldn't almost-always need to hold one of them.